x-pack/plugins/reporting/server/export_types/csv_searchsource/generate_csv/generate_csv.ts

```diff
@@ -360,19 +365,21 @@ export class CsvGenerator {
 
       // Add warnings to be logged
       if (this.csvContainsFormulas && escapeFormulaValues) {
-        warnings.push(
-          i18n.translate('xpack.reporting.exportTypes.csv.generateCsv.escapedFormulaValues', {
-            defaultMessage: 'CSV may contain formulas whose values have been escaped',
-          })
-        );
+        warnings.push(i18nTexts.escapedFormulaValuesMessage);
       }
     } catch (err) {
       this.logger.error(err);
       if (err instanceof KbnServerError && err.errBody) {
         throw JSON.stringify(err.errBody.error);
       }
+
       if (err instanceof esErrors.ResponseError && [401, 403].includes(err.statusCode ?? 0)) {
-        throw new AuthenticationExpiredError();
+        reportingError = new AuthenticationExpiredError();
+        warnings.push(
+          totalRecords > 0 ? i18nTexts.partialResultsMessage : i18nTexts.noResultsMessage
```

Comment:
I'm not sure we need to have an extra message to state there were no results due to the authentication error. The `partialResultsMessage` seems suitable to me for both cases. It risks us showing some false positives, since there could be other explanations for the CSV not having any results.

Reply:
I gave this a more thought, the primary issue for me is that if a user downloads a CSV that we say has partial results but then contains no results it could create some confusion.

However, this will only ever occur when a user sends a report request that either contains an expired auth header (unlikely) or that expires before it gets back the first set of results (also unlikely, there is probably something bigger wrong).

Happy to remove the extra i18n for now.